smFISH_utils/metrics/visualisation.py

```diff
@@ -0,0 +1,228 @@
+import matplotlib.pyplot as plt
+import matplotlib.gridspec as gridspec
+
+
+def show_cell_types(latent_seq, labels_seq, latent_fish, labels_fish, title='latent.svg', x_lim=[-85, 85],
+                    y_lim=[-85, 85]):
```

Comment:
Would it work to make all these smfish-specific function methods in the `VariationalInferenceFish` class?

Reply:
That would keep all the smFISH-specific code together in one place.